README.md

```diff
@@ -30,145 +30,144 @@ Styleguide for low-fidelity paper prototyping and wireframing.
 - Images
 - Modal dialogs
 
-3.2. _MUST_. For **navigation bars**, when the bar is prefixed by a trademark or logo, include the word `LOGO` at the beggining of the bar.
+3.2. _MUST_ include the word `LOGO` at the beginning of **navigation bars** prefixed by a trademark or logo.
```

Comment:
Since we both agree these changes somehow make sense. And considering we'll eventually segment the text by topics. Isn't it better if we just do it now, instead of reverting back all the changes that are already made by this PR?